core-ui/src/shared/helpers/getResourceUrl.js

```diff
@@ -3,8 +3,20 @@ export function getResourceUrl() {
   const fullResourceApiPath = queryParams.get('fullResourceApiPath');
   const resourceApiPath = queryParams.get('resourceApiPath');
 
+  // resource names can't be lowercased - Role Bindings there could be name with upper case
+  const tokens = window.location.pathname.split('/');
+  let tokensLength = tokens.length;
+
+  if (tokensLength === 3) tokensLength = 2;
+  else if (tokensLength === 5) tokensLength = 4;
```

Comment:
what about number 7? 🙃 
we have a longer path for example here: `namespaces/NamespaceName/customresourcedefinitions/details/CRDName/v1alpha1/CRName`
Also, we don't lowercase just the last resourceName in this case, shouldn't we apply it to all in the path (so, in the example above - NamespaceName, CRDName and CRName)?